### ISSUE TITLE
Pre-processor definitions make this library a bit unfriendly to others who don't use cmake

### ISSUE THREAD

**Issue by KennethThompson** [SamVanheer/AngelscriptUtils#2]
- This is just a nit-pick, but it took me a half hour to figure out that I was missing some preprocessor definitions, namely: AS_STRING_OBJNAME=\"string\"  You might consider putting this information in the readme for the project.  

- Also, you might consider breaking out your project into 1. a static lib with your AngelscriptUtils code, 2. an angelscript project that simply links to the angelscript repo, and 3. an executable project with the example base classes and tester functions.
 
- I don't use cmake as I'm a former windows developer and prefer having an IDE crutch (codeblocks).  Also, while we are on the subject of cmake, you used cmake version 3.6.  Ubuntu's apt-get is stuck on version 3.5.2.  I was forced to upgrade to get cmake to test compile the project.  Not a huge deal, but still a road block for newbies.

**Comment by SamVanheer** [comment]
I apologize for the late response, i've been a bit busy of late.

I've added a note to the README to inform users of the macro, thanks for the suggestion.

I'll consider reworking the project architecture when i have some more time to work on this.

I used CMake 3.6 to use VS_STARTUP_PROJECT, but that has never worked out well for me, so i can downgrade it to an older version, provided no other CMake statements require 3.6.
